### PR TITLE
doubledown changes

### PR DESCRIPTION
**Pull request by blackpatches81**

### CODE REVIEW

File: src/main/java/models/Player.java [hunk 1]
```diff
@@ -65,5 +66,9 @@ public void winBet(){
         money = money + bet;
     }
 
+    public void doubleDown(int setBet)
+    {
+        money = (setBet * 2);
+    }
```
Comment (NathanHealea) [inline src/main/java/models/Player.java]: Double down doesn't need to take a int. All it should is double bet and store it back into bet.

File: src/test/java/models/testPlayer.java [hunk 2]
```diff
@@ -121,6 +121,14 @@ public void testSplit(){
...
 
     }
+    @Test
+    public void testDoubleDown(){
+        Player p = new Player();
+        p.setBet(250);
+        p.doubleDown(300);
+        assertNotEquals(250, p.bet);
+    }
+
```
Comment (NathanHealea) [inline src/test/java/models/testPlayer.java]: Method does not check the correct functionality of the method.

File: src/main/java/models/Player.java [hunk 3]
```diff
@@ -65,5 +66,9 @@ public void winBet(){
         money = money + bet;
     }
 
+    public void doubleDown(int setBet)
```
Comment (foulgerd) [inline src/main/java/models/Player.java]: doubleDown shouldn't require any int to be passed to it.

File: src/main/java/models/Player.java [hunk 4]
```diff
@@ -65,5 +66,9 @@ public void winBet(){
...
+    {
+        money = (setBet * 2);
```
Comment (foulgerd) [inline src/main/java/models/Player.java]: All you need to do is multiply bet by two and assign that to bet again

File: src/test/java/models/testPlayer.java [hunk 5]
```diff
@@ -121,6 +121,14 @@ public void testSplit(){
 
 
     }
+    @Test
+    public void testDoubleDown(){
+        Player p = new Player();
+        p.setBet(250);
+        p.doubleDown(300);
+        assertNotEquals(250, p.bet);
```
Comment (foulgerd) [inline src/test/java/models/testPlayer.java]: I would change this test to test for the expected bet value. This way, we can see that you're getting the correct value.

**Pull request closed without merging**